topdown/http_test.go

```diff
@@ -129,22 +130,31 @@ func TestHTTPGetRequestTlsInsecureSkipVerify(t *testing.T) {
 
 	resultObj := ast.MustInterfaceToValue(expectedResult)
 
+	type httpsStruct struct {
+		note     string
+		rules    []string
+		expected interface{}
+		//expectedError error
```

Comment:
`expectedError` seems wasn't used anyways.

Reply:
yes, it is not used.
the go-lint will error if it is not used any place.

Reply:
Ok, let's remove it then.

Reply:
OK, rebase it later.